eng/config/rulesets/Shipping.ruleset

```diff
@@ -55,8 +56,11 @@
     <Rule Id="CA1823" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/20404: Enable rule CA1823 (remove unused field) once we have a fixer for it -->
     <Rule Id="CA1824" Action="None" /> <!-- mark assemblies with NeutralResourcesLanguageAttribute -->
     <Rule Id="CA1827" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/37959: DoNotUseCountWhenAnyCanBeUsed -->
+    <Rule Id="CA1834" Action="None" /> <!-- Consider using 'StringBuilder.Append(char)' when applicable -->
+    <Rule Id="CA1835" Action="None" /> <!-- Prefer the 'Memory'-based overloads for 'ReadAsync' and 'WriteAsync' -->
     <Rule Id="CA2000" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/25880 -->
     <Rule Id="CA2007" Action="Warning" />
+    <Rule Id="CA2100" Action="None" /> <!-- AD0001 ReviewSqlQueriesForSecurityVulnerabilities: https://github.com/dotnet/roslyn-analyzers/issues/3613 -->
```

Comment:
https://github.com/dotnet/roslyn-analyzers/pull/3614 should be merged and this line should be reverted. This issue is a bug in the underlying options infrastructure in the analyzers repo, which can show up in any rule which uses an analyzer config option.